app/scripts/controllers/preferences.js

```diff
@@ -69,12 +72,25 @@ export default class PreferencesController {
       ...opts.initState,
     };
 
+    ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+    initState.useTokenDetection = Boolean(process.env.TOKEN_DETECTION_V2);
+    ///: END:ONLY_INCLUDE_IN
+
+    this.network = opts.network;
     this._onInfuraIsBlocked = opts.onInfuraIsBlocked;
     this._onInfuraIsUnblocked = opts.onInfuraIsUnblocked;
     this.store = new ObservableStore(initState);
     this.store.setMaxListeners(13);
     this.tokenListController = opts.tokenListController;
 
+    ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+    this.handleMmiPortfolio = opts.handleMmiPortfolio;
+
+    if (!process.env.IN_TEST) {
+      this.mmiConfigurationStore = opts.mmiConfigurationStore.getState();
```

Comment:
True we can add documentation to these 2 methods, also we have this MMI Portfolio logic `(handleMmiPortfolio)` in here because every time an address gets added, removed or they are synced, the extension needs to tell that to the portfolio app, and since those operations are done here, we call this logic in here as well. But we can improve the naming of the method to better fit this controller semantic.